.cirrus.yml

```diff
@@ -210,8 +210,8 @@ build_each_commit_task:
 
     setup_environment_script: '$SCRIPT_BASE/setup_environment.sh |& ${TIMESTAMP}'
     build_each_commit_script:
-        - 'source $SCRIPT_BASE/lib.sh'
-        - 'git fetch --depth $CIRRUS_CLONE_DEPTH origin $CIRRUS_BASE_BRANCH |& ${TIMESTAMP}'
+        - 'source $SCRIPT_BASE/lib.sh &> /dev/null'
+        - 'git fetch --depth $CIRRUS_CLONE_DEPTH origin $DEST_BRANCH |& ${TIMESTAMP}'
         - 'make build-all-new-commits GIT_BASE_BRANCH=origin/$CIRRUS_BASE_BRANCH |& ${TIMESTAMP}'
```

Comment:
In IRC you wrote <i>I don't think $CIRRUS_BASE_BRANCH gets set after a PR merges.</i> Shouldn't this instant also get updated?

Reply:
Woops, missed that, thanks.  Yet another case for "just do this all in a script".  I'll look into that tomorrow.

Reply:
okay got it.